L10nSnapshotTests/L10nSuite2SnapshotTests.swift

```diff
@@ -123,17 +123,27 @@ class L10nSuite2SnapshotTests: L10nBaseSnapshotTests {
         // First time only: The message "Your passwords are now protected
         // by Face ID..." is present when the Firefox app is run after the
         // simulator has been erased.
-        waitForExistence(app.navigationBars.element(boundBy: 0), timeout: 3)
-        waitForExistence(app.otherElements.buttons.element(boundBy: 2))
-        app.otherElements.buttons.element(boundBy: 2).tap()
+        // The biometric screen is supposed to appear here.
+        // https://github.com/mozilla-mobile/firefox-ios/issues/15642
+        // waitForExistence(app.navigationBars.element(boundBy: 0), timeout: 3)
+        // waitForExistence(app.otherElements.buttons.element(boundBy: 2))
+        // app.otherElements.buttons.element(boundBy: 2).tap()
 
         let passcodeInput = springboard.secureTextFields.firstMatch
         waitForExistence(passcodeInput, timeout: 30)
         passcodeInput.tap()
         passcodeInput.typeText("foo\n")
 
+        // The biometric screen is not supposed to be here during the first run.
+        // This snippet is here as a workaround.
+        // https://github.com/mozilla-mobile/firefox-ios/issues/15642
+        waitForNoExistence(passcodeInput)
+        if app.otherElements.buttons.staticTexts.element(boundBy: 5).exists {
```

Comment:
A workaround for locating the "Continue" button without relying on the exact text.

Reply:
The biometric screen is an overlay of something else, so the indexes include the buttons that have been covered. 😬